spicedb-as-library/main.go

```diff
@@ -106,7 +113,7 @@ func newServer(ctx context.Context) (server.RunnableServer, error) {
 			return ctx, nil
 		}),
 		server.WithHTTPGateway(util.HTTPServerConfig{HTTPEnabled: false}),
-		server.WithMetricsAPI(util.HTTPServerConfig{HTTPEnabled: true}),
+		server.WithMetricsAPI(util.HTTPServerConfig{HTTPEnabled: false}),
```

Comment:
It took me ages to figure out that if this is `true` but the port is already being used, the server stops suddenly! (Feels like a bug in SpiceDB)